test/e2e/experiment/experiment_test.go

```diff
@@ -1046,3 +1063,65 @@ func createGoalID(t *testing.T) string {
 	}
 	return fmt.Sprintf("%s-goal-id-%s", prefixTestName, newUUID(t))
 }
+
+// This helper tries to stop the running experiments
+// that are finished testing and waiting for deletion.
+// This will improve the load on the http-stan while analysing the other experiments
+// speeding up and improve timeout flaky tests.
+// Since this is optional, it will ignore any errors.
+func stopExperiment(
```

Comment:
As described in the comments, I added this helper to force the running experiments to stop once the test is finished. This will decrease the number of analyses in the experiment calculator.